aries-site/src/data/structures/components.js

```diff
@@ -1413,7 +1413,7 @@ export const components = [
   },
   {
     name: 'Spinner',
-    accessibility: 'Failed WCAG 2.2 AA',
+    accessibility: 'Passed WCAG 2.2 AA',
```

Comment:
Should we update this to say passed AAA, it looks like we don't have any rules failing
```suggestion
    accessibility: 'Passed WCAG 2.2 AAA',
```